docs/controls/ImageEx.md

```diff
@@ -29,6 +29,7 @@ The [ImageEx Control](https://docs.microsoft.com/dotnet/api/microsoft.toolkit.uw
 | IsCacheEnabled | bool | Gets or sets a value indicating whether gets or sets cache state |
 | ImageExCachingStrategy | enum | Gets or sets a value indicating how the Image will be cached |
 | CornerRadius | double | Get or set the radius of image corner |
+| EnableLazyLoading | bool | Gets or sets a value indicating whether gets or sets is lazy loading enable. (17763 or higher supported) |

```

Comment:
This is weird: "**Gets or sets** a value indicating whether **gets or sets**"

Reply:
Done, change to **Gets or sets a value indicating is lazy loading enabled. (17763 or higher supported)**